### PR TITLE
[one-cmds] Introduce one-partition

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce one-partition command tool.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #8570, from draft #9074

File: compiler/one-cmds/one-partition [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    config = configparser.ConfigParser()
+    # make option names case sensitive
+    config.optionxform = str
+    parsed = config.read(config_path)
+    if not parsed:
+        raise FileNotFoundError('Not found given configuration file')
+    if not config.has_section(section):
+        raise AssertionError("configuration file doesn't have '" + section + "' section")
+    for key in config[section]:
+        print("!!! key", key, " section", section, " value", config[section][key])
```
Comment (mhs4670go) [inline compiler/one-cmds/one-partition:94]: Maybe for degugging?

File: compiler/one-cmds/one-partition [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    if not _utils._is_valid_attr(args, 'part_file'):
+        missing.append('part_file')
+    if not _utils._is_valid_attr(args, 'input_file'):
+        missing.append('input_file')
+    if len(missing):
+        parser.error('the following arguments are required: ' + ' '.join(missing))
+    return
+
+
+def _parse_cfg_and_overwrite(config_path, section, args):
```
Comment (mhs4670go) [inline compiler/one-cmds/one-partition:76]: This function exists in `utils.py`. Is there any reason of not using it?

File: compiler/one-cmds/one-partition (hunk 1 above)
Comment (seanshpark) [reply compiler/one-cmds/one-partition:94]: Ah, yes, I'll remove this.

File: compiler/one-cmds/one-partition (hunk 2 above)
Comment (seanshpark) [reply compiler/one-cmds/one-partition:76]: Ah, just copy-paste without deep review -_-;

File: compiler/one-cmds/one-partition [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+def main():
+    # parse arguments
+    parser = _get_parser()
+    args = _parse_arg(parser)
+
+    # parse configuration file
+    _utils._parse_cfg(args, 'one-partition')
+
+    print(args)
```
Comment (mhs4670go) [inline compiler/one-cmds/one-partition:118]: This one is also for debugging?

File: compiler/one-cmds/one-partition [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    if _utils._is_valid_attr(args, 'config'):
+        config_path = getattr(args, 'config')
+        _utils._parse_cfg_and_overwrite(config_path, 'one-partition', args)
+
+    print(args)
```
Comment (mhs4670go) [inline compiler/one-cmds/one-partition:123]: ditto

**Review by mhs4670go — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
@mhs4670go , thank you for the review!

**Pull request merged**